Makefile

```diff
@@ -19,7 +19,6 @@ install-docker:
 	install -D -m 644 docker-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
 	if [ ! -f ${SYSCONFDIR}/${DOCKER}-storage-setup ]; then \
 		install -D -m 644 docker-storage-setup-override.conf ${SYSCONFDIR}/${DOCKER}-storage-setup; \
-		echo "CONTAINER_THINPOOL=docker-pool" >> ${SYSCONFDIR}/${DOCKER}-storage-setup; \
```

Comment:
If we are not changing upstream default, there is no need to change this. right? In fact I think it probably is a good idea to also add `STORAGE_DRIVER=devicemapper` to make default explicit. Downstream should overwrite this file to change upstream default.

Reply:
Lets leave it untouched for now. I can't see how it is related to STORAGE_DRIVER=auto